tecken/tests/test_libdockerflow.py

```diff
@@ -18,14 +18,13 @@ def test_check_storage_urls_happy_path():
 
 
 def test_check_storage_urls_missing(settings):
-    symbol_urls = [
-        "http://s3.example.com/public",
-        "http://s3.example.com/other-bucket",
-    ]
-    normal_storage = SymbolStorage(symbol_urls)
+    symbol_storage = SymbolStorage(
+        upload_url="http://s3.example.com/public",
+        download_urls=["http://s3.example.com/other-bucket"],
+    )
```

Comment:
According to `pytest_django`'s docs for the `settings` pytest fixture:
> A Django settings object which restores changes after the testrun

Could an alternative here be to change the `settings` that `symbol_storage` pulls from ordinarily and call `symbol_storage(force_recreate=True)` as with the `tecken/tests/test_download.py` tests?

I don't think one way is better than the other, just curious...

Reply:
I think that would work as well. The tests are rather inconsistent overall, and so far I haven't spent time trying to fix that. I probably should at some point.